lib/balance-tx/cardano-balance-tx.cabal

```
@@ -99,35 +99,67 @@ test-suite test
   import:             language, opts-exe
   ghc-options:        -with-rtsopts=-M2G -with-rtsopts=-N4
   type:               exitcode-stdio-1.0
-  hs-source-dirs:     test/spec
+  hs-source-dirs:     test/spec test/data
```

This doesn't seem needed
```suggestion
  hs-source-dirs:     test/spec
```

---

@Anviking 

Thanks for spotting this -- I'll make a follow-up PR with this fix in it.